examples/pennylane/3_Quantum_chemistry_with_VQE.ipynb

```diff
@@ -235,7 +235,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Put the descriptive comments above rather than below the corresponding code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>